### PR TITLE
Adds missing request body documentation.

### PR DESCRIPTION
**Pull request by mkraft**
#### Summary

Adds missing documentation for request body.

#### Ticket Link

https://github.com/mattermost/mattermost-api-reference/issues/655

### CODE REVIEW

File: v4/source/saml.yaml [hunk 1]
```diff
@@ -27,6 +27,15 @@
...
       operationId: GetSamlMetadataFromIdp
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                saml_metadata_url:
+                  type: string
+                  description: The URL from which to retrieve the Saml IDP data.
```
Comment (cwarnermm) [inline v4/source/saml.yaml:38]:
```suggestion
                  description: The URL from which to retrieve the SAML IDP data.
```

**Review by cwarnermm — APPROVED** [review]
Thanks, @mkraft. One small capitalization issue identified. Approving for expediency.

**Review by catalintomai — APPROVED** [review]

File: v4/source/saml.yaml (hunk 1 above)
Comment (mkraft) [reply v4/source/saml.yaml:38]: This is an acronym (not an initialism), just confirming... do we use all caps for those?

Comment (mkraft) [reply v4/source/saml.yaml]: Thanks. I guess I was thinking of [British English](https://en.wikipedia.org/wiki/Capitalization_in_English#Capitalization_of_acronyms_and_initialisms).

**Pull request merged**